src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import core.basesyntax.objects.Ball;
+
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        final int numberOfBalls = 3;
+        final int maxNumberOnTheBall = 100;
+        Ball[] balls = new Ball[numberOfBalls];
```

Comment:
it's not critical, but we can avoid creating array and just create objects and print them